doc/src/build/cli-client.md

```diff
@@ -0,0 +1,602 @@
+| `gas`                 | Obtain all gas objects owned by the address.                                                                                                                                                                                                  |
+| `help`                | Print this message or the help of the given subcommand(s).                                                                                                                                                                                    |
+| `merge-coin`          | Merge two coin objects into one coin.                                                                                                                                                                                                         |
+| `new-address`         | Generate new address and keypair with keypair scheme flag {ed25519 or secp256k1 or secp256r1} with optional derivation path, default to m/44'/784'/0'/0'/0' for ed25519 or m/54'/784'/0'/0/0 for secp256k1 or m/74'/784'/0'/0/0 for secp256r1 |
+| `new-env`             | Add new Sui environment.                                                                                                                                                                                                                      |
+| `object`              | Get object information.                                                                                                                                                                                                                       |
+| `objects`             | Obtain all objects owned by the address.                                                                                                                                                                                                      |
+| `pay`                 | Pay SUI to recipients following specified amounts, with input coins. Length of recipients must be the same as that of amounts.                                                                                                                |
+| `pay_all_sui`         | Pay all residual SUI coins to the recipient with input coins, after deducting the gas cost. The input coins also include the coin for gas payment, so no extra gas coin is required.                                                          |
+| `pay_sui`             | Pay SUI coins to recipients following specified amounts, with input coins. Length of recipients must be the same as that of amounts. The input coins also include the coin for gas payment, so no extra gas coin is required.                 |
+| `profile-transaction` | Profile the gas usage of a transaction.  Outputs a file `gas_profile_{tx_digest}_{unix_timestamp}.json` which can be opened in a flamegraph tool such as speedscope.                                                                          |
```

Comment:
(Needs to be updated now that the output file is configurable -- maybe mention the given format is the default?).